content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -80,43 +92,61 @@ Once the credit note has been posted, a :guilabel:`Credit Notes` smart button wi
 Generate coupons from a ticket
 ==============================
 
-Coupons can be used to alter the price of products or orders. The usage constraints of a coupon are
-defined by conditional rules. *Coupon Programs* are configured in the *Sales* or *Website*
-applications.
+Coupons can be used to alter the price of products or orders. Conditional rules define the usage
+constraints of a coupon. *Coupon Programs* are configured in the *Sales*, *Point of Sale*, or
+*Website* applications.
 
-.. note::
-   The *eCommerce* module must be installed in order to create coupon codes from the *Website*.
+.. important::
+   The *eCommerce* module must be installed to create coupon codes from the *Website*.
 
 To generate a coupon, open a *Helpdesk* ticket and click on the :guilabel:`Coupon` button in the
-upper left corner. Select an option from the :guilabel:`Coupon Program` drop-down menu, then click
-:guilabel:`Generate`.
+top-left corner. Select an option from the :guilabel:`Coupon Program` drop-down menu.
 
 .. image:: after_sales/after-sales-generate-coupon.png
    :align: center
    :alt: View of a coupon generation window.
 
-The :guilabel:`Coupon Code` can be copied directly from the pop-up window (by clicking the
-:guilabel:`Copy` button), or sent in an email by clicking :guilabel:`Send`.
+.. note::
+   To create a new :guilabel:`Coupon Program`, navigate to :menuselection:`Sales app --> Products
+   --> Discount & Loyalty` and click :guilabel:`New`. To make the program available to share with
+   *Helpdesk* customers, the :guilabel:`Program Type` **must** be set to :guilabel:`Coupons`. This
+   generates single-use coupon codes that grant immediate access to rewards and discounts.
+
+  Coupon programs can also be created in the *Point of Sale* application or *Website* application.
+  More information on creating discount and loyalty programs can be found :doc:`here
+  </applications/sales/sales/products_prices/loyalty_discount>`.
+
```

Comment:
I remember someone editing a doc of mine saying we should use the doc name (or close to it) instead of "this doc" or "here". Just an idea- not necessary.